src/Umbraco.Web.UI.Client/src/views/logviewer/overview.controller.js

```diff
@@ -74,7 +81,7 @@
                         "query": "Not(@Level='Verbose') and Not(@Level='Debug')"
                     },
                     {
-                        "name": "Find all logs that has an exception property (Warning, Error & Critical with Exceptions)",
+                        "name": "Find all logs that has an exception property (Warning, Error & Fatal with Exceptions)",
```

Comment:
@nathanwoulfe or @AndyButland 
Suggest that the saved JS config file on disk of daved searches is updated as well please `src/Umbraco.Web.UI/config/logviewer.searches.config.js`

Reply:
Done in latest commits